### PR TITLE
(feat): create user and another tables models and migration

### PR DESCRIPTION
**Pull request by Depesi**
Created models and migration for 6 tables - Users, UserChat, ChatRoom, Message, EventStatusHistory, ChangeInfos. Creted controller and some endpoints for Users table.

### CODE REVIEW

File: src/models/changeinfos.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class changeInfos extends Model {
```
Comment (VadymKharyton) [inline src/models/changeinfos.js:5]: What about change this to Pascal case

Comment (Depesi) [reply src/models/changeinfos.js:5]: this file was created automatically, if we need to change this, so after each generated modelwill need to change it, are you sure that need to change it?

Comment (VadymKharyton) [reply src/models/changeinfos.js:5]: We should support one style in each part in application.
In this case, all models needs to be defined in one style, in Pascal case.
I don't think that it's a big problem to rerun `model:generate` again with same params, and regenerate these few files

Comment (Depesi) [reply src/models/changeinfos.js:5]: So just regenerate to ChangeInfos and anothers file regenerate same with PascalCase?

Comment (VadymKharyton) [reply src/models/changeinfos.js:5]: Yes. If there are any other models with camelCase or lowercase, please regenerate them as well

Comment (Depesi) [reply src/models/changeinfos.js]: Done

File: src/models/changeinfos.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const {
```
Comment (VadymKharyton) [inline src/models/changeinfos.js:1]: I see, that some of files with models are named in lowercase. That's incorrect, they should be in camelCase.
Use this [page](https://daily-dev-tips.com/posts/git-basics-help-my-case-sensitive-filename-changes-dont-commit/) to correctly rename these files and track them in git

Comment (VadymKharyton) [reply src/models/changeinfos.js:1]: Okay, we decided to not do that.

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**